src/_includes/partials/components/filters.njk

```diff
@@ -0,0 +1,93 @@
+{% set filterableOptions = { projects: [], types: [], topics: [], publishers: [] } %}
+{% for resource in collections['resources_' + lang] %}
+    {% set filterableOptions = filterableOptions | objectArrayPush("projects", resource.data.projects) %}
+    {% set filterableOptions = filterableOptions | objectArrayPush("types", resource.data.type) %}
+    {% set filterableOptions = filterableOptions | objectArrayPush("topics", resource.data.topics) %}
+    {% set filterableOptions = filterableOptions | objectArrayPush("publishers", resource.data.publisher) %}
+{% endfor %}
+<div id="filters" class="filters flow">
+    <h3>{% __ 'filters' %}</h3>
+    <div class="flow">
+        {% if collections['topics_' + lang] | length > 1 %}
```

Comment:
I think here you should be checking the length of `filterableOptions.topics`, because these filters would still show if there are more than 1 topics in the CMS even if they aren't assigned to any resources.